cockroachcloud/console-access-management.md

```diff
@@ -45,6 +45,84 @@ A Console Admin is an all-access role. A Console Admin can perform the following
 - [Restore databases and tables from a {{ site.data.products.db }} backup](backups-page.html#ways-to-restore-data)
 - [Delete an organization](#delete-an-organization)
 
+## Service accounts
+
+{% include_cached cockroachcloud/experimental-warning.md %}
+
+Service accounts are used by applications accessing the [Cloud API](cloud-api.html) to manage {{ site.data.products.db }} clusters within the organization. Service accounts are not for human users.
+
+To create a service account:
+
+1. Click **Create Service Account**.
+1. In the **Create service account** dialog:
+    1. Enter the **Account name**.
+    1. (Optional) Enter a **Description** of the service account.
+    1. Set the **Permissions** of the service account.
+
+        Granting `ADMIN` permissions allows the service account full authorization for the organization, where the service account can create, modify, and delete clusters.
+
+        The `CREATE` permission allows the service account to create new clusters within the organization.
+
+        The `DELETE` permission allows the service account to delete clusters within the organization.
+
+        The `EDIT` permission allows the service account to modify clusters within the organization.
+
+        The `READ` permission allows the service account to get details about clusters within the organization.
+
+    1. Click **Create**.
+
+1. [Create an API key](#create-api-keys) for the newly created service account, or click **Skip** to go back to the Service Accounts table.
+
+### Modify a service account
+
+To modify the name, description, or permissions of a service account:
+
+1. Click the **Action** button for the service account name in the **Service Accounts** table.
+1. Select **Edit**.
+1. In the **Edit service account** dialog, modify the name, description, or permissions for the service account.
+1. Click **Save changes**.
+
+### API access
+
+{% include_cached cockroachcloud/experimental-warning.md %}
+
+Each service account can have one or more API keys. API keys are used to authenticate and authorize service accounts when using the API. All API keys created by the account are listed under **API Access**.
+
+We recommend creating service accounts with the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege), and giving each application that accesses the API its own API key. This allows fine-grained permission and API key access to the cluster.
+
+#### Create API keys
+
+To create an API key:
+
```

Comment:
Add step to get to the Service Account Details page first here, same for Delete/Edit an API key